pubspec.yaml

```diff
@@ -24,7 +25,7 @@ flutter:
   plugin:
     platforms:
       android:
-        package: com.instabug.instabugflutter
+        package: com.instabug.flutter
```

Comment:
This change might affect people importing the plugin in native code only, which could be less than 1% of our users. We can mention it in our Changelog for reference.